docs/src/main/sphinx/release/release-424.md

```diff
@@ -0,0 +1,46 @@
+# Release 424 (16 Aug 2023)
+
+## General
+
+* Improve query performance for large Trino clusters with many worker nodes. ({issue}`18542`)
```

Comment:
This PR doesn't improve query performance. It improves coordinator background overhead that's unrelated to query execution.

Reply:
Does it need a release note, then?